test/commit-split.ts

```diff
@@ -182,6 +182,21 @@ describe('CommitSplit', () => {
     });
   }
 
+  // Test valid CommitSplitOptions.packagePaths combinations.
+  // Intentionally inconsistent trailing slashes to test path normalization.
+  it('validates configured paths on / separator', () => {
+    const cs = new CommitSplit({
+      packagePaths: ['two-three', 'one-two', 'three/', 'one', 'one-two-three'],
```

Comment:
Would it be worth adding a `foo/bar` test case, since you mention this was one of the problems in issue?